constants/user-constant.js

```diff
@@ -5,8 +5,11 @@ module.exports = Object.freeze({
     ACTIVE: 'Active',
     DISABLED: 'Disabled',
     DELETED: 'Deleted',
+    DISABLED: 'Disabled',
     // user role
     ADMIN: 'Admin',
     MEMBER: 'Member',
-    NON_MEMBER: 'Non-Member'
+    NON_MEMBER: 'Non-Member',
+    // Login Event
+    LOGIN: 'Login'
```

Comment:
The Login constant is already defined in the submodule, we should avoid having it defined in two different places